mesonbuild/modules/gnome.py

```diff
@@ -269,12 +269,12 @@ class VapiTarget(build.CustomTarget):
 # https://bugzilla.gnome.org/show_bug.cgi?id=774368
 gresource_dep_needed_version = '>= 2.51.1'
 
-native_glib_version: T.Optional[str] = None
-
 class GnomeModule(ExtensionModule):
 
     INFO = ModuleInfo('gnome')
 
+    native_glib_version: T.ClassVar[T.Optional[str]] = None
```

Comment:
This seems wrong. Moving it to a class var is not the only way to avoid it being a global, we could also make it be... an instance var. Like the other instance vars we already have.

Am I missing something that means it has to be a classvar specifically?

Reply:
I think historically we had one instance per subproject. But now we should always have only one instance of module.